cmake/sysbuild/sdp.cmake

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# Update eGPIO images KConfig based on SDP configuration.
+#
+# Usage:
+#   egpio_update_kconfig()
+#
+function(egpio_update_kconfig)
+  if(SB_CONFIG_EGPIO_BACKEND_MBOX)
+    foreach(image ${PRE_CMAKE_IMAGES})
+      set_config_bool(${image} CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX y)
+      set_config_bool(${image} CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG n)
+    endforeach()
```

Comment:
Wouldn't setting `CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX` be enough? It is a choice config, so, if I understand correctly, enabling `CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX` should disable `CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG`